shared/common-adapters/clickable-box.js.flow

```diff
@@ -2,10 +2,12 @@
 import React, {Component} from 'react'
 
 export type Props = {
+  children?: any,
   style?: Object,
-  onClick: () => void,
-  hoverColor?: ?string,
-  backgroundColor?: ?string,
+  onClick?: ?() => void,
```

Comment:
re: styling the mobile use-case is just to make something clickable with a onPress color, and this does that